### PR TITLE
fix: caniuse-lite is outdated` warning while running server #4338

### PR DESCRIPTION
**Pull request by aman-earthian**
Fixes # 4343

#### Short description of what this resolves:
Updated caniuse-lite.

#### Changes proposed in this pull request:

-updates caniuse lite to v1.0.30001048 in response to issue #4338

### CODE REVIEW

**Comment by snitin315** [comment]
Please use Semantic PR naming.

**Comment by snitin315** [comment]
also ~~Fixes # 4343~~ Fixes #4338

File: package.json [hunk 1]
```diff
@@ -184,4 +184,4 @@
     "node-fetch",
     "ua-parser-js"
   ]
-}
+}
```
Comment (iamareebjamal) [inline package.json:187]: revert

**Comment by aman-earthian** [comment]
@snitin315, done

File: package.json (hunk 1 above)
Comment (aman-earthian) [reply package.json:187]: can't understand what you are trying to point out.

**Comment by kushthedude** [comment]
Already fixed!

**Pull request closed without merging**